package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "cloudflare-worker-test-harness",
+  "version": "0.0.1",
+  "main": "index.js",
+  "dependencies": {
+    "jest-fetch-mock": "^1.6.5",
+    "whatwg-fetch": "^2.0.4"
+  },
+  "scripts": {
+    "build": "webpack --config webpack.config.js",
```

Comment:
Not that important but this can be just `webpack` as it use `webpack.config.js` by default.

Reply:
Sorry I didn't see the `entry` is not using the default. Please ignore this comment.

Reply:
Do you need webpack at all?